### DISCUSSION TITLE
Is there a way to make the slide zoom on click instead of hover?

### DISCUSSION THREAD

**Discussion by beckj94** [express-labs/pure-react-carousel#448] [Q&A]
I want to zoom in on the image on click instead of zoom because this default behavior is kinda jarring.

**Comment by Conor-Hinchee** [comment]
@beckj94 thanks for asking this in Q&A.

So the short answer is out of the box no. 

The longer answer is that you could roll your own using the `<Image />` component and pass down the necessary event handlers and styles or create your own Component that is a child of each Slide.  

Hope this helps.
